src/portable/renesas/rusb2/dcd_rusb2.c

```diff
@@ -519,12 +558,17 @@ static void process_bus_reset(uint8_t rhport)
     ++ctr;
   }
   tu_varclr(&_dcd);
+
+#if (CFG_TUSB_RHPORT1_MODE & OPT_MODE_DEVICE)
+  dcd_event_bus_reset(rhport, TUSB_SPEED_HIGH, true);
+#else
   dcd_event_bus_reset(rhport, TUSB_SPEED_FULL, true);
+#endif
 }
 
 static void process_set_address(uint8_t rhport)
 {
-  const uint32_t addr = RUSB2->USBADDR_b.USBADDR;
+  const uint32_t addr = RUSB2->USBADDR & 0xFF;
```

Comment:
Please let me know why this change needed.
I am concerned to fit for RX running as big-endian.

Reply:
ah, great reminder. I am updating this PR with this branch with more changes https://github.com/hathach/tinyusb/tree/renesas_ra_hs_rebased 
I will try to double check big-endian as well as try to see if I could compile with big-endian mode for testing the changes.

Reply:
Agree, this should be reverted